data/.gitignore

```diff
@@ -1 +1,21 @@
 regression
+
+# ORCA
```

Comment:
I don't see why we need to ignore so many files here. If you run calcs in these directories locally, perhaps ignore them only locally, see https://git-scm.com/docs/gitignore for details.